### ISSUE TITLE
config.web_console.templates_path is not picked up

### ISSUE THREAD

**Issue by lustremedia** [rails/web-console#141]
I'd like to style the console from being positioned 'fixed' to being positioned 'relative'. I have tried to pull in styles as mentioned in the ReadMe but style.css is never picked up, or does not overwrite the default styles.

Rails 4.2

**Comment by gsamokovarov** [comment]
Can you give us the exact locations of the stylesheet, its content and the config you set?

**Comment by lustremedia** [comment]
First I tried it in my application "app/assets/stylesheets/web_console" and the config in application.rb. After that did not work I thought it might conflict with the assets pipeline and moved it to the location set in the README "app/views/web_console" no luck there either.

**Comment by sh19910711** [comment]
I guess it may be fine to use `config.web_console.template_paths` instead of `config.web_console.templates_path` (and it seems to need a fix for the code or the doc).

https://github.com/rails/web-console/blob/26753c6279db73260a7de018e5630b0b5d56cfe1/lib/web_console/railtie.rb#L43

**Comment by gsamokovarov** [comment]
Oh, there is a typo in the README. Let me fix that @lustremedia, can you set the setting `config.web_console.template_paths` (instead of `templates_path`) and try again? Thanks @sh19910711  for noticing that.

**Comment by lustremedia** [comment]
"config.web_console.template_paths" worked for me! However, is the console always appended to the end of the page? I want the console to be somewhere in my content flow. How can I accomplish that?

**Comment by gsamokovarov** [comment]
@lustremedia You can't do that at the moment. We may apply a smarter strategy in the future that appends the console in the head or at the end of the body of the current content so it also results in a valid HTML document as well. Sadly, this isn't the case right now.

**Comment by lustremedia** [comment]
@gsamokovarov No Problem. What would be really nice: A button where you can hide or show the console so that it does not interfere with the layout!

**Comment by gsamokovarov** [comment]
See #140, its already in the making. :-)

**Comment by lustremedia** [comment]
Ah cool!

**Comment by DannyBen** [comment]
Hey guys - piggybacking on this thread since its the same problem, maybe the docs could be clearer - I am still unable to make it work.

I have added this (all variations) in my `config/development.rb`
```
  config.web_console.templates_path = 'app/views/web_console'
  config.web_console.template_path = 'app/views/web_console'
  config.web_console.template_paths = 'app/views/web_console'
```

And in my `app/views/web_console` folder, I have tried placing any of these files
* style.css (as the README said)
* styles.css.erb (as appears in the templates source folder)
* styles.css (just another guess)
* styles.scss (and another guess)

Restarted the server, but the style does not seem to be picked up. 
I know the config is working, since I have also placed `config.web_console.whitelisted_ips` in it


BTW - I think `template_paths` is the least logical variation out of the three. Its a single path, so should be `template_path` or `templates_path` at most.

**Comment by gsamokovarov** [comment]
Thanks for taking the time to write this down @DannyBen, much appreciated. The correct one is `config.template_paths`. You have to end your style file `erb` like so: `style.css.erb`.

> BTW - I think template_paths is the least logical variation out of the three. Its a single path, so should be template_path or templates_path at most.

This makes sense. I'm open to changing it, as long as we don't break the backwards compatibility with the old flag.

**Comment by DannyBen** [comment]
Hi @gsamokovarov - well, the name of the property is less critical than the fact it does not seem to work. Either I have done something wrong (and I think I have done whatever the documentation said), or there may be a bug.

I assumed I can take any file that exists in the templates source folder, and put it in MY template folder, and it will be picked up. This is my interpretation of the README instructions.

So I placed `app/views/web_console/styles.css.erb`, and changed some obvious style in it, restarted the server just in case, and still nothing.

**Comment by anandbait** [comment]
@DannyBen Were you able to solve your issue?
Were you able to override styles for the console?

**Comment by DannyBen** [comment]
> @DannyBen Were you able to solve your issue?
> Were you able to override styles for the console?

This was some 9 years ago, I do not remember... :)
Most likely I was unable to solve it, otherwise I would have probably posted here.

**Comment by anandbait** [comment]
@DannyBen  Thanks for letting me know. I am facing this issue too.
@gsamokovarov I am not able to get custom styles working.
In my application.rb I have set 
```
config.web_console.template_paths = 'app/views/web_console'
```
And I have created `app/views/web_console/style.css.erb`
I have tried overriding styles of `.console` and just to be sure I have used `!important` on styles.
But it is not reflecting.

I checked through the code and found that styles are being rendered from 
```
render_inlined_string('style')
```
Which is calling [this code](https://github.com/rails/web-console/blob/f3d437ce89339dde5adaa7adad809405b1df9277/lib/web_console/view.rb#L34)
```
    def render_inlined_string(template)
      render(template: template, layout: "layouts/inlined_string")
    end
```
I don't see any magical happening here.
How are overridden styles picked? Am I missing anything?
